pkg/controller/controller.go

```diff
@@ -777,7 +777,7 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 		if respCode == http.StatusAccepted {
 			glog.V(5).Infof("Received asynchronous provisioning response for Instance %v/%v of ServiceClass %v at Broker %v: response: %v", instance.Namespace, instance.Name, serviceClass.Name, brokerName, response)
 			if response.Operation != "" {
-				instance.Status.LastOperation = response.Operation
+				instance.Status.LastOperation = &response.Operation
```

Comment:
Where does the store back into etcd happen? I see the call to `c.updateInstanceCondition()` but I'm assuming that just updates the Condition stuff and not the Status in etcd. If the etcd update of Status happens further back up the stack, outside of this current func, then we may indeed have a problem if `response` is gc'd and if golang doesn't track that we have a pointer to some of its data. In this test program: https://play.golang.org/p/EKefLEo03C when I turn on/off line 23 I do see go's reuse of memory change - meaning when I set `f` to nil it appears to reuse memory, but when I don't set it to nil then it skips that chunk of memory - so maybe it does track it.  Just hard to know for sure.

For example, see https://golang.org/pkg/runtime/#KeepAlive  I think they talk about a similar issue and use KeepAlive to resolve it.

I'm probably worrying for nothing, but wanted to mention it just in case someone knows for sure how go deals with this.  Of course see my other comment on why this change is technically not needed at all :-)